cumulusci/tasks/github/merge_freeze.py

```diff
@@ -0,0 +1,20 @@
+from cumulusci.tasks.github.base import BaseGithubTask
+
+
+class EnableMergeFreeze(BaseGithubTask):
+
+    task_options = {
+        "repos": {
+            "description": (
+                "The list of owner, repo key pairs for which to generate release notes."
+                + " Ex: 'owner': SalesforceFoundation 'repo': 'NPSP'"
+            ),
+            "required": True,
+        },
+    }
+
+    def _run_task(self):
+        for project in self.options["repos"]:
+            if project["owner"] and project["repo"]:
```

Comment:
Do you want to handle an `else` clause and notify the user that the provided repo isn't going to be processed because it's missing owner/repo? Should it fail outright if this condition is met?